src/FeedForwardPid.cpp

```diff
@@ -6,27 +6,45 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#include <limits>
+
 #include "iarc7_motion/FeedForwardPid.hpp"
 
 using namespace Iarc7Motion;
 
-FeedForwardPid::FeedForwardPid(double p_gain, double i_gain, double d_gain) : p_gain_(p_gain), i_gain_(i_gain), d_gain_(d_gain), setpoint_(0.0)
+FeedForwardPid::FeedForwardPid(double p_gain, double i_gain, double d_gain,
+                               double i_accumulator_max, double i_accumulator_min)
+    : p_gain_(p_gain),
+      i_gain_(i_gain),
+      d_gain_(d_gain),
+      setpoint_(0.0),
+      i_accumulator_(0.0),
+      last_current_value_(0.0),
+      i_accumulator_max_(i_accumulator_max),
+      i_accumulator_min_(i_accumulator_min)
 {
     // Nothing to do
 }
 
 double FeedForwardPid::update(double current_value, double time_delta)
 {
-
-    static double i_accumulator{0.0};
-    static double last_current_value{0.0};
-
+    if (time_delta == 0) {
+        time_delta = std::numeric_limits<double>::min();
```

Comment:
I disagree.  It's not impossible physically, because the granularity on your clock is finite, so if the value changes before your clock updates then you get the input changing with a time delta of 0.  In that case, we should really assume the time delta is approximately the clock granularity.  It would make sense to log that this is happening though, probably as a warning.  I would also like to avoid being dependent on our time library's implementation, so what we could do is have a parameter passed in for the granularity of the clock.